### DISCUSSION TITLE
Multiple domains for server and client interactions

### DISCUSSION THREAD

**Discussion by nitchimon** [projectdiscovery/interactsh#167] [Q&A] [answered]
The situation I have is that there are multiple internal and external facing domains.

Each server may be on various domains.

Example:
mydomain.com
myotherdomain.com
someotherdomain.com

If I start the  interactsh-server  with a domain -domain mydomain.com  There seems to be no way to have it also listen in on the other 2 or more domains.
If I remove the -domain argument,  the server and client do not interact and the interact-server stops listening altogether on dns and http ports.

Is there a way to allow it to work with multiple domains ?

Thanks

**Comment by ehsandeep** [comment, marked as answer]
@nitchimon currently, only a single server is supported at one time, but this will change in future updates as we are facing issues using a single server for public uses and integrations, so the pool of servers will be supported in coming updates.

**Reply by nitchimon** [reply]
Thank you !
Client works with one server at a time.

But can the interactsh-server listen and work with multiple domains ?

thanks again.

**Reply by ehsandeep** [reply]
No, the support of multiple servers can be only implemented at the client level, it's not possible to set up multiple domains from the same host at the server-side, you will need to have multiple unique interacts servers separately that you can use with the client as detailed here - https://github.com/projectdiscovery/interactsh/issues/170

**Reply by nitchimon** [reply]
Thank you for the reply !
